src/averageAges.js

```diff
@@ -36,8 +56,23 @@ function calculateMenAverageAge(people, century) {
  *
  * @return {number}
  */
+function findLostChild(people, mother) {
```

Comment:
findLostChild suggests that the function returns the found child, but your function returns simply whether or not the mother has child, so it is better to name it smth like hasChild